components/Heading/Heading.js

```diff
@@ -3,30 +3,33 @@ import ArrowBackIcon from "../../public/icons/Chevron_left";
 import LoginIcon from "../../public/icons/login";
 import { useRouter } from "next/router";
 
-export default function Heading({ children, previousLevelUrl, isShowButton }) {
+export default function Heading({
+  children,
+  previousLevelUrl,
+  isHideBackButton,
+}) {
   const router = useRouter();
 
-  if (!router.isReady) {
-    return <h1>loading</h1>;
-  }
+  /* if (!router.isReady) {
+    return <p>loading</p>;
+  } */
 
   function handleGoBackPage() {
     router.push(previousLevelUrl);
   }
-
   return (
     <StyledHeader>
       <StyledButton
         type="button"
         onClick={handleGoBackPage}
         aria-label="go back to the last page"
-        isShowButton={isShowButton}
+        isHideBackButton={isHideBackButton}
       >
-        <ArrowBackIcon />
+        <ArrowBackIcon color={`var(--red-vine-color)`} />
       </StyledButton>
       <h1>{children}</h1>
       <StyledButton type="button" aria-label="log in" isShowButton={true}>
-        <LoginIcon />
+        <LoginIcon color={`var(--red-vine-color)`} />
```

Comment:
i am not sure why there is a login icon. maybe remove it as long as users can not login?

Reply:
🤭 I keep it there, because Paul didn't notice it. But you are right. I shouldn't be so lazy. I remove it. 😸